frontend/src/static/js/v_summary.js

```diff
@@ -223,12 +224,17 @@ window.vSummary = {
           }
         });
       });
-      const creationDateArray = window.app.creationDate.split(' ');
-      const year = creationDateArray.pop();
-      creationDateArray.pop();
-      const maxDate = window
-          .moment(`${creationDateArray.join(' ')} ${year}`, 'ddd MMM D HH:mm:ss YYYY')
-          .format('YYYY-MM-DD');
+
+      if (window.app.untilDate) {
+        maxDate = window.app.untilDate;
+      } else {
+        const creationDateArray = window.app.creationDate.split(' ');
+        const year = creationDateArray.pop();
+        creationDateArray.pop();
+        maxDate = window
+            .moment(`${creationDateArray.join(' ')} ${year}`, 'ddd MMM D HH:mm:ss YYYY')
+            .format('YYYY-MM-DD');
```

Comment:
Hmm... instead of doing this, perhaps we can just add the values to summary.json such that the `window.app.untilDate` will always be set. Better maintainability and one less library to import.